DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: shiny.telemetry
 Type: Package
 Title: 'Shiny' App Usage Telemetry
-Version: 0.1.0.9003
+Version: 0.2.0
```

Comment:
There is a version mismatch (0.1.1 in NEWS and branch name and 0.2.0 here). I'm okay with using 0.2.0, which includes a new feature - MS SQL Server support, but I'm not forcing it.

Reply:
Agree, resolved! Sorry about the mismatch